sdk/cognitiveservices/azure-cognitiveservices-inkrecognizer/azure/cognitiveservices/inkrecognizer/_models.py

```diff
@@ -0,0 +1,722 @@
+        If this InkDrawing is an alternate, returns None.
+
+        :rtype: List[InkDrawing] or None
+        """
+
+        return self._alternates
+
+
+class Line(InkRecognitionUnit):
+    """
+    A Line instance represents the collection of one or more ink strokes that
+    were recognized as a line.
+    """
+    _ink_recognition_unit_kind = InkRecognitionUnitKind.LINE
+
+    def __init__(self, json_object):
+        InkRecognitionUnit.__init__(self, json_object)
+        self._recognized_text = json_object.get("recognizedText", "")
+        self._alternates = self._parse_alternates(json_object.get("alternates", []))
+
+    def _parse_alternates(self, alternates):  # pylint:disable=no-self-use
+        return [alternate["recognizedString"] for alternate in alternates]
+
+    @property
+    def alternates(self):
+        # type: () -> List[str]
+        """
+        A list of alternate strings reported by the service.
+
+        :rtype: List[str]
+        """
+        return self._alternates
+
+    @property
+    def recognized_text(self):
+        # type: () -> str
+        """
+        The recognized string of the line. If the words in the line are not
+        recognized, an empty string is returned.
+
+        :rtype: str
+        """
+        return self._recognized_text
+
+    @property
+    def bullet(self):
+        # type: () -> Optional[InkBullet, None]
+        """
+        If the line has a bullet, return it as InkBullet. Otherwise, return None.
+        One line can have at most one bullet.
+
+        :rtype: InkBullet or None
+        """
+
+        for unit in self.children:
+            if unit.kind == InkRecognitionUnitKind.INK_BULLET:
+                return unit
+        return None
+
+    @property
+    def words(self):
+        # type: () -> List[InkWord]
+        """ All the words in this line.
+
+        :rtype: List[InkWord]
+        """
+        return [unit for unit in self.children if unit.kind == InkRecognitionUnitKind.INK_WORD]
+
+
+class Paragraph(InkRecognitionUnit):
+    """
+    A Paragraph instance represents the collection of one or more ink strokes
+    that were recognized as a paragraph.
+    """
+    _ink_recognition_unit_kind = InkRecognitionUnitKind.PARAGRAPH
+
+    @property
+    def recognized_text(self):
+        # type: () -> str
+        """
+        The recognized string of the paragraph. If the words in the paragraph
+        are not recognized, an empty string is returned.
+
+        :rtype: str
+        """
+        return "\n".join([child.recognized_text
+                          for child in self.children])
+
+    @property
+    def lines(self):
+        # type: () -> List[Line]
+        """
+        All the lines in the paragraph.
+
+        :rtype: List[Line]
+        """
+        return [unit for unit in self.children
+                if unit.kind == InkRecognitionUnitKind.LINE]
+
+    @property
+    def list_items(self):
```

Comment:
I would suggest using the name `listitems` since we are generally using `list_<something>` to indicate a service methods that lists resources.

Reply:
Other interfaces are like "ink_words", "writing_regions", etc. So I think it's better to keep the consistency for list_items.